config_example.sh

```
@@ -1,287 +1,545 @@
+################################################################################
+## Network Settings
+##
+
+# IP_STACK - IP stack for the cluster.
+# Default: "v6"
+# Choices: "v4", "v6", "v4v6"
+#
+#export IP_STACK=v4
+
+# HOST_IP_STACK - IP stack for the hosts. 
+# Default: If unset, defaults to ${IP_STACK}.
+# Choices: "v4", "v6", "v4v6"
+# You can set IP_STACK to "v4" or "v6" and HOST_IP_STACK to "v4v6" to install a 
+# single-stack cluster on dual-stack hosts.
+#
+#export HOST_IP_STACK=v4v6
+
+# NTP_SERVERS - Configure custom ntp servers if needed
+# Default is unset.
+#
+#export NTP_SERVERS="00.my.internal.ntp.server.com;01.other.ntp.server.com"
+
+# PROVISIONING_NETWORK_PROFILE - Provisioning network profile
+# Default is "Managed".
+#export PROVISIONING_NETWORK_PROFILE=Managed
+
+# Provisioning interface on the helper node
 #export PRO_IF="eno1"
 
 # Internal interface
 #export INT_IF="eno2"
 
-# Provisioning interface within the cluster
+# Provisioning interface within the cluster (Default: "enp1s0")
 #export CLUSTER_PRO_IF="eno1"
 
-# Which disk to deploy
-#export ROOT_DISK_NAME="/dev/sda"
-
-# Cluster name
-#export CLUSTER_NAME="mycluster"
-
-# Domain name
-#export BASE_DOMAIN="kni.lab.metal3.io"
-
 # Manage bridge
 #export MANAGE_BR_BRIDGE=n
 
-# Path to the json files with ipmi credentials
-#export NODES_FILE="/root/git/dev-scripts/bm.json"
+# Enable virtualMediaViaExternalNetwork in provisioning configuration.
+# More info: https://github.com/openshift/cluster-baremetal-operator/blob/master/api/v1alpha1/provisioning_types.go#L163-L171
+#export ENABLE_VIRTUAL_MEDIA_VIA_EXTERNAL_NETWORK=true
 
-# Whether the installation is on baremetal or not
-#export NODES_PLATFORM=BM
+# Set the DHCP lease time (Default: 60)
+#export DHCP_LEASE_EXPIRY=60
 
-# DNS_VIP
+# DNS_VIP (Defaults to v6)
+#
+# v4 Default:
 #export DNS_VIP="11.0.0.2"
+#
+# v6 Default:
+#export DNS_VIP="fd2e:6f44:5dd8:c956::2"
 
-# Network type
+# Network type (Defaults to v6)
```

The `defaults to v6` here and elsewhere is slightly misleading - the values for `NETWORK_TYPE`, and subnet/VIP values are [defaulted depending on the value](https://github.com/openshift-metal3/dev-scripts/blob/master/network.sh#L104..L130) of `IP_STACK`

---

That's true these values are based off `IP_STACK` and `HOST_IP_STACK`. 🤔 I will update the wording.